filebeat/harvester/log.go

```diff
@@ -64,17 +63,9 @@ func (h *Harvester) Harvest() {
 		ts, text, bytesRead, jsonFields, err := readLine(reader)
 		if err != nil {
 			if err == errFileTruncate {
-				seeker, ok := h.file.(io.Seeker)
-				if !ok {
-					logp.Err("can not seek source")
-					return
-				}
-
 				logp.Info("File was truncated. Begin reading file from offset 0: %s", h.Path)
```

Comment:
That is a good point. We should make a WARN out of this and add a comment that this means that not all content was read potentially. WDYT?

But I suggest to make this in a second PR, ok?